### ISSUE TITLE
Footnote location on the page with Urban Institute boilerplate

### ISSUE THREAD

**Issue by awunderground** [UrbanInstitute/rmarkdown-factsheets#9]
Footnotes appear after the Urban Institute contact information but appear after the Urban Institute boilerplate. I'm unsure if this should change for the boilerplate.

**Comment by fblackshaw** [comment]
For consistency with other Urban pubs, it would be great if the footnotes were above the boilerplate so the copyright statement was the absolute last text on page 2. On a scale of 1 (can definitely live without) to 10 (absolutely must have), I'd rank this a 7 (would really like to [try to] have).

**Comment by awunderground** [comment]
This request scared me! It is resolved with 294bef7c8d52415b9a5d6f60616af2340ae593c0

The first and second pages have different top and bottom margins. The margins for the first page are created in the YAML header: `geometry: "left=1in,right=1in,top=0.35in,bottom=0.6in"`. The margins for the last page are created in the .Rmd:
```
\newpage{}
\newgeometry{left=1in,right=1in,top=1in,bottom=1.5in}
```

If the fact sheets increase to three or more pages, we'll need a separate set of margins for the middle pages. I could create something like `\urbannewpage{}` and `\urbanlastpage{}`
